src/pages/admin/import-video.tsx

```diff
@@ -409,6 +451,16 @@ class IndexApp extends React.Component<Props, State> {
             </div>
         </Modal>
     }
+    renderDeleteVideo() {
+        return <Modal isOpen={this.state.showDeleteVideo}>
+            <div>
+                <div>Enter ID: <input value={this.state.toDeleteVideo} onChange={(item: any) => this.setState({ toDeleteVideo: item.target.value })} /></div>
+                <div>Type &quot;Delete forever&quot;: <input value={this.state.deleteConfirmation} onChange={(item: any) => this.setState({ deleteConfirmation: item.target.value })} /></div>
```

Comment:
If they've manually typed out the ID of the video isn't this a bit redundent?

Reply:
Yeah, I suppose it's pretty difficult to type the ID accidently. Thanks for the suggestions!